modules/util/portal-tools-bundle-support/src/test/java/com/liferay/portal/tools/bundle/support/commands/BundleSupportCommandsTest.java

```diff
@@ -546,8 +577,7 @@ public void handle(HttpExchange httpExchange) throws IOException {
 	}
 
 	private static URL _getHttpServerUrl(String contextPath) throws Exception {
-		return new URL(
-			"http", "localhost.localdomain", _HTTP_SERVER_PORT, contextPath);
+		return new URL("http", "127.0.0.1", _HTTP_SERVER_PORT, contextPath);
```

Comment:
Oops I left these in my commit.  I had to change this to get it to work on my machine. Do you want to change it back?

Reply:
Yes, please change it back and resend

Reply:
done and sent. https://github.com/petershin/liferay-portal/pull/1025